pkg/auth/user/util.go

```diff
@@ -3,9 +3,16 @@ package user
 import (
 	"sort"
 
+	"github.com/gogo/protobuf/proto"
 	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/auth/permissions/utils"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/stackrox/rox/pkg/logging"
+	"go.uber.org/zap"
 )
 
+var utilLog = logging.LoggerForModule()
```

Comment:
Rename the parameter from `log` to `logger` within `LogSuccessfulUserLogin` and name this `log`? Since there's no other log within the package, don't see why it needs to be named explicitly here.